### ISSUE TITLE
Glibc 2.34+ wont have SIGSTKSZ

### ISSUE THREAD

**Issue by kraj** [abseil/abseil-cpp#957]
when you wll compile abseil-cpp with glibc/master ( future 2.34 ) then it will fail with this msg

```
/mnt/b/yoe/master/build/tmp/work/riscv64-yoe-linux/abseil-cpp/20210324+gitAUTOINC+e1d388e7e7-r0/recipe-sysroot-native/usr/bin/riscv64-yoe-linux/riscv64-yoe-linux-clang++ -Dfailure_signal_handler_EXPORTS -I/mnt/b/yoe/master/build/tmp/work/riscv64-yoe-linux/abseil-cpp/20210324+gitAUTOINC+e1d388e7e7-r0/git -target riscv64-yoe-linux      -mlittle-endian -mno-relax -Qunused-arguments -fstack-protector-strong  -O2 -D_FORTIFY_SOURCE=2 -Wformat -Wformat-security -Werror=format-security  --sysroot=/mnt/b/yoe/master/build/tmp/work/riscv64-yoe-linux/abseil-cpp/20210324+gitAUTOINC+e1d388e7e7-r0/recipe-sysroot  -O2 -pipe -g -feliminate-unused-debug-types -fmacro-prefix-map=/mnt/b/yoe/master/build/tmp/work/riscv64-yoe-linux/abseil-cpp/20210324+gitAUTOINC+e1d388e7e7-r0=/usr/src/debug/abseil-cpp/20210324+gitAUTOINC+e1d388e7e7-r0                      -fdebug-prefix-map=/mnt/b/yoe/master/build/tmp/work/riscv64-yoe-linux/abseil-cpp/20210324+gitAUTOINC+e1d388e7e7-r0=/usr/src/debug/abseil-cpp/20210324+gitAUTOINC+e1d388e7e7-r0                      -fdebug-prefix-map=/mnt/b/yoe/master/build/tmp/work/riscv64-yoe-linux/abseil-cpp/20210324+gitAUTOINC+e1d388e7e7-r0/recipe-sysroot=                      -fdebug-prefix-map=/mnt/b/yoe/master/build/tmp/work/riscv64-yoe-linux/abseil-cpp/20210324+gitAUTOINC+e1d388e7e7-r0/recipe-sysroot-native=   -fvisibility-inlines-hidden -target riscv64-yoe-linux      -mlittle-endian -mno-relax -Qunused-arguments -fstack-protector-strong  -O2 -D_FORTIFY_SOURCE=2 -Wformat -Wformat-security -Werror=format-security  --sysroot=/mnt/b/yoe/master/build/tmp/work/riscv64-yoe-linux/abseil-cpp/20210324+gitAUTOINC+e1d388e7e7-r0/recipe-sysroot -fPIC -Wall -Wextra -Wcast-qual -Wconversion -Wfloat-overflow-conversion -Wfloat-zero-conversion -Wfor-loop-analysis -Wformat-security -Wgnu-redeclared-enum -Winfinite-recursion -Wliteral-conversion -Wmissing-declarations -Woverlength-strings -Wpointer-arith -Wself-assign -Wshadow -Wstring-conversion -Wtautological-overlap-compare -Wundef -Wuninitialized -Wunreachable-code -Wunused-comparison -Wunused-local-typedefs -Wunused-result -Wvla -Wwrite-strings -Wno-float-conversion -Wno-implicit-float-conversion -Wno-implicit-int-float-conversion -Wno-implicit-int-conversion -Wno-shorten-64-to-32 -Wno-sign-conversion -DNOMINMAX -std=gnu++14 -MD -MT absl/debugging/CMakeFiles/failure_signal_handler.dir/failure_signal_handler.cc.o -MF absl/debugging/CMakeFiles/failure_signal_handler.dir/failure_signal_handler.cc.o.d -o absl/debugging/CMakeFiles/failure_signal_handler.dir/failure_signal_handler.cc.o -c /mnt/b/yoe/master/build/tmp/work/riscv64-yoe-linux/abseil-cpp/20210324+gitAUTOINC+e1d388e7e7-r0/git/absl/debugging/failure_signal_handler.cc
/mnt/b/yoe/master/build/tmp/work/riscv64-yoe-linux/abseil-cpp/20210324+gitAUTOINC+e1d388e7e7-r0/git/absl/debugging/failure_signal_handler.cc:139:24: error: no matching function for call to 'max'
  size_t stack_size = (std::max(SIGSTKSZ, 65536) + page_mask) & ~page_mask;
                       ^~~~~~~~
/mnt/b/yoe/master/build/tmp/work/riscv64-yoe-linux/abseil-cpp/20210324+gitAUTOINC+e1d388e7e7-r0/recipe-sysroot/usr/lib/riscv64-yoe-linux/11.1.0/../../../include/c++/11.1.0/bits/algorithmfwd.h:407:5: note: candidate template ignored: deduced conflicting types for parameter '_Tp' ('long' vs. 'int')
    max(const _Tp&, const _Tp&);
    ^
/mnt/b/yoe/master/build/tmp/work/riscv64-yoe-linux/abseil-cpp/20210324+gitAUTOINC+e1d388e7e7-r0/recipe-sysroot/usr/lib/riscv64-yoe-linux/11.1.0/../../../include/c++/11.1.0/bits/stl_algo.h:3467:5: note: candidate template ignored: could not match 'initializer_list<type-parameter-0-0>' against 'long'
    max(initializer_list<_Tp> __l, _Compare __comp)
    ^
/mnt/b/yoe/master/build/tmp/work/riscv64-yoe-linux/abseil-cpp/20210324+gitAUTOINC+e1d388e7e7-r0/recipe-sysroot/usr/lib/riscv64-yoe-linux/11.1.0/../../../include/c++/11.1.0/bits/stl_algo.h:3461:5: note: candidate function template not viable: requires single argument '__l', but 2 arguments were provided
    max(initializer_list<_Tp> __l)
    ^
/mnt/b/yoe/master/build/tmp/work/riscv64-yoe-linux/abseil-cpp/20210324+gitAUTOINC+e1d388e7e7-r0/recipe-sysroot/usr/lib/riscv64-yoe-linux/11.1.0/../../../include/c++/11.1.0/bits/algorithmfwd.h:412:5: note: candidate function template not viable: requires 3 arguments, but 2 were provided
    max(const _Tp&, const _Tp&, _Compare);
    ^
1 error generated.
```

this is because `SIGSTKSZ` definition is changed

https://sourceware.org/git/?p=glibc.git;a=commit;h=6c57d320484988e87e446e2e60ce42816bf51d53

**Comment by kraj** [comment]
its fixed via a9831f1cb please backport it to lts branch too

**Comment by derekmauro** [comment]
https://github.com/abseil/abseil-cpp/releases/tag/20210324.2 has been released to address this issue.
